slice-test/slice-mapper-test/src/test/groovy/com/cognifide/slice/mapper/annotation/ChildrenTest.groovy

```diff
@@ -0,0 +1,73 @@
+package com.cognifide.slice.mapper.annotation
+
+import com.cognifide.slice.test.setup.BaseSetup
+import org.junit.Assert
+
+/**
+ * @author Mariusz Kubiś Date: 14.04.15
+ */
+class ChildrenTest extends BaseSetup {
```

Comment:
This test is only testing happy scenarios. Please add some edge cases, especially if resource is not found or points to not existing resource.